home06/product.py

```diff
@@ -0,0 +1,14 @@
+from pydantic import BaseModel, Field
+
+
+class Product(BaseModel):
+    product_id: int
+    product_title: str = Field(..., title='Title', max_length=50)
+    product_description: str = Field(..., title='Description')
+    product_price: int = Field(..., title='Price', gt=0)
+
+
+class ProductIn(BaseModel):
+    product_title: str = Field(..., title='Title', max_length=50)
+    product_description: str = Field(..., title='Description')
+    product_price: int = Field(..., title='Price', gt=0)
```

Comment:
Модели почти идентичны, значит можно одну наследовать от другой
```suggestion
class ProductIn(BaseModel):
    product_title: str = Field(..., title='Title', max_length=50)
    product_description: str = Field(..., title='Description')
    product_price: int = Field(..., title='Price', gt=0)
    
    
class Product(ProductIn):
    product_id: int
```

Reply:
Кстати, модели `pydantic` можно было разместить в одном файле